.github/workflows/ci.yml

```diff
@@ -14,10 +12,14 @@ jobs:
           node-version: '14'
           cache: 'yarn'
       - run: yarn install --frozen-lockfile
+      - run: yarn build
       - run: yarn test
 
   deploy:
-    needs: build
+    on:
+      push:
```

Comment:
If we add `workflow_dispatch` here we can trigger deploys from github manually. If the AWS work will mean this becomes redundant then just ignore this comment.